SPECS/policycoreutils/policycoreutils.spec

```diff
@@ -274,22 +282,13 @@ The policycoreutils-restorecond package contains the restorecond service.
 %{_mandir}/ru/man8/sepolicy-transition.8*
 %{_mandir}/ru/man8/sepolicy.8*
 
-%files
-%license policycoreutils/COPYING
+%files min
 %{_sbindir}/restorecon
-%{_sbindir}/restorecon_xattr
-%{_sbindir}/fixfiles
```

Comment:
Is fixfiles not a core functionality? From what I understand it's running restorecon under the hood but I thought it was needed to label system on boot.

Reply:
No, it's labeled by the build process (images) or install process (iso).

Reply:
https://github.com/microsoft/CBL-Mariner/blob/1.0/toolkit/tools/imagegen/installutils/installutils.go#L1519